client/src/components/HostMenu.jsx

```diff
@@ -16,6 +16,14 @@ const HostMenu = ({ guests, menu, setTopLevelState }) => {
     });
   });
 
+  useEffect(() => {
+    socket.emit('joinRoom', { sessionId });
+  });
+
+  useEffect(() => {
+    socket.emit('hostJoined', { sessionId });
+  });
```

Comment:
Same with these, should be able to add the enpty array and they will run once on join.
```suggestion
  useEffect(() => {
    socket.emit('joinRoom', { sessionId });
  }, []);

  useEffect(() => {
    socket.emit('hostJoined', { sessionId });
  }, []);
```

Reply:
Added!